apps/music/js/metadata/vorbis_picture.js

```diff
@@ -0,0 +1,131 @@
+/* global BlobView */
+/* exported VorbisPictureComment */
+'use strict';
+
+var VorbisPictureComment = (function() {
+  /**
+   * Read the picture frame from the block blob
+   */
+  function readPicFrame(view) {
+    // Source of the structure:
+    // https://xiph.org/flac/format.html#metadata_block_picture
+    // http://flac.sourceforge.net/format.html#metadata_block_picture
+    //
+    var kind = view.readUnsignedInt();
+    if (kind != 3) {
+      return null;
+    }
+    var len = view.readUnsignedInt();
+    var mimetype = view.readASCIIText(len);
+    if (mimetype == '-->') {
+      // XXX we don't support yet URL
```

Comment:
Can you log something to the console if we hit this? It'll help a lot in case someone expects this feature to work; then when we ask for a logcat, we'll know they're using URLs for their art and we can tell them it's not supported.